src/tbb/allocator.cpp

```diff
@@ -67,11 +85,15 @@ static void  (*deallocate_handler)(void* pointer) = nullptr;
 //! Initialization routine used for first indirect call via cache_aligned_allocate_handler.
 static void* initialize_cache_aligned_allocate_handler(std::size_t n, std::size_t alignment);
 
+#if !defined(__TBB_USE_MSVC_ALIGNED_MALLOC)
 //! Allocates memory using standard malloc. It is used when scalable_allocator is not available
 static void* std_cache_aligned_allocate(std::size_t n, std::size_t alignment);
+#endif
 
+#if !defined(__TBB_USE_POSIX_MEMALIGN) && !defined(__TBB_USE_MEMALIGN) && !defined(__TBB_USE_MSVC_ALIGNED_MALLOC)
 //! Allocates memory using standard free. It is used when scalable_allocator is not available
 static void  std_cache_aligned_deallocate(void* p);
+#endif
```

Comment:
What about the approach like in `std_cache_aligned_allocate`?
```suggestion
#if !defined(__TBB_USE_MSVC_ALIGNED_MALLOC)
//! Allocates memory using standard malloc. It is used when scalable_allocator is not available
static void* std_cache_aligned_allocate(std::size_t n, std::size_t alignment);

#if !defined(__TBB_USE_POSIX_MEMALIGN) && !defined(__TBB_USE_MEMALIGN)
//! Allocates memory using standard free. It is used when scalable_allocator is not available
static void  std_cache_aligned_deallocate(void* p);
#endif
#endif // !defined(__TBB_USE_MSVC_ALIGNED_MALLOC)
```

Reply:
Done.